include/fmt/printf.h

```diff
@@ -578,8 +578,32 @@ struct printf_context {
     std::back_insert_iterator<Buffer>, typename Buffer::value_type> type;
 };
 
-typedef basic_format_args<printf_context<internal::buffer>::type> printf_args;
-typedef basic_format_args<printf_context<internal::wbuffer>::type> wprintf_args;
+typedef printf_context<internal::buffer>::type printf_char_context;
+typedef printf_context<internal::wbuffer>::type printf_wchar_context;
```

Comment:
`printf_char_context` and `printf_wchar_context` should be called `printf_context` and `wprintf_context` for consistency with `format_context` and `wformat_context`. The old `printf_context` template should be renamed to `basic_printf_context`.

Reply:
I ended up using `basic_printf_context_t` because it caused a name conflict with the inner typedef. That naming style is also with the format context

    template <typename OutputIt, typename Char = char>
    //using format_context_t = basic_format_context<OutputIt, Char>;
    struct format_context_t { typedef basic_format_context<OutputIt, Char> type; };

I hope that is ok with you.